libsolidity/formal/ModelCheckerSettings.h

```diff
@@ -26,6 +26,31 @@
 namespace solidity::frontend
 {
 
+struct ModelCheckerContracts
+{
+	/// By default all contracts are analyzed.
+	static ModelCheckerContracts Default() { return {}; }
+
+	static std::optional<ModelCheckerContracts> fromString(std::string const& _contracts);
```

Comment:
Nit: now I know what this is doing, splits up the `<path>:<contract>,<path>:<contract>...` string. Perhaps add it as a comment?